backend/.prettierrc

```
@@ -1 +1,8 @@
-{}
+{
+  "semi": false,
```

```suggestion
  "semi": true,
```
The airBNB style guide is a react style guide, but you should try to keep some consistencies between your front and back to make context switching easier. 

You may want to adjust your prettier configuration based on whats going to be enforced in the front